packages/examples/cvat/exchange-oracle/tests/integration/services/test_exchange.py

```diff
@@ -457,10 +462,12 @@ def test_create_assignment_in_validated_and_rejected_job(self):
         ):
             manifest = json.load(data)
             mock_get_manifest.return_value = manifest
-            assignment_id = create_assignment(cvat_project_1.id, user_address)
+            assignment_id = create_assignment(cvat_project_1.id, previous_user.wallet_address)
+            assert assignment_id is None  # should not assign job to the same user more than once
```

Comment:
I suggest to add another test for this scenario.

Reply:
Can't imagine how separate test will be beneficial. It would've make sense if this scenario tested something different, but it in fact tests the opposite behaviour to the new one. 

So making changes the only thing that made sense.

I'm happy to add a new test case, but just want to get the clear picture of how it should look like and how should it be different from this one.

Reply:
Currently, from what I can see, this test checks that:
- a previous assignee cannot be assigned
- a new assignee can be assigned

In this PR we're covering a new requirement, so I suggest that there is a separate test, focused on this. The benefit is that we'll have a test checking this requirement specifically, it will be clear that this is a requirement from the test name and not from a comment.

Reply:
A bit verbose to my taste, but I guess it makes sense indeed. Split these test-cases in two.